### PR TITLE
distribute e2e jobs  better

### PR DESCRIPTION
**Pull request by gero3**
**Description**
e2e jobs now get distributed in blocks based on file name which has the disadvantage that job 7 takes longer since those contain the bigger examples. This PR loops through the examples and distributes them evenly.
So like the following scheme:
JOB 0: Example 0, Example 8, Example 16, ...
JOB 1: Example 1, Example 9, Example 17, ...
JOB 2: Example 2, Example 10, Example 18, ...
JOB 3: Example 3, Example 11, Example 19, ...
JOB 4: Example 4, Example 12, Example 20, ...
JOB 5: Example 5, Example 13, Example 21, ...
JOB 6: Example 6, Example 14, Example 22, ...
JOB 7: Example 7, Example 15, Example 23, ...

This PR also makes the total amount of jobs configurable from ci.yml by adding the environment variable JOBS.

### CODE REVIEW

File: examples/webgl_shadowmap_progressive.html [hunk 1]
```diff
@@ -101,7 +101,7 @@
 				scene.add( groundMesh );
 
 				// model
-				function loadModel() {
+				async function loadModel() {
```
Comment (Mugen87) [inline examples/webgl_shadowmap_progressive.html:104]: The changes to `webgl_shadowmap_progressive` seem unrelated to the PR.

Comment (gero3) [reply examples/webgl_shadowmap_progressive.html:104]: webgl_shadowmap_progressive takes about 5 minutes on its own in the CI system which was why job 7 took 7 minutes and the others about 2-3 minutes. I was testing this example because of this.
